includes/agentic-commerce/class-wc-stripe-agentic-commerce-product-list-table.php

```
@@ -0,0 +1,425 @@
+			/* translators: %s: number of products. */
+			$message = _n( '%s product included in the Agentic Commerce catalog sync.', '%s products included in the Agentic Commerce catalog sync.', $count, 'woocommerce-gateway-stripe' );
+		} else {
+			return;
+		}
+		// phpcs:enable WordPress.Security.NonceVerification.Recommended
+
+		printf(
+			'<div class="notice notice-success is-dismissible"><p>%s</p></div>',
+			esc_html( sprintf( $message, number_format_i18n( $count ) ) )
+		);
+	}
```

PHPMD (2.15.0)

[error] render_bulk_action_notice accesses the super-global variable $_GET. (undefined)

(Superglobals)